source/features/one-click-review-submission.tsx

```diff
@@ -43,18 +42,19 @@ function replaceCheckboxes(originalSubmitButton: HTMLButtonElement): void {
 			classes.push('tooltipped tooltipped-nw tooltipped-no-delay');
 		}
 
+		const label = document.querySelector(`label[for="${radio.id}"]`)?.textContent;
```

Comment:
The labels are not adjacent siblings anymore, so I'm just grabbing the text from their actual `<label>`s.

Let me know if there's a way to do this that's more in keeping with the repo